programs/contractors/contractors.json

```diff
@@ -57,5 +57,32 @@
         "url": "https://github.com/json-schema-org/JSON-Schema-Test-Suite"
       }
     ]
+  },
+  {
+    "name": "Jason Desrosiers",
+    "github": "jdesrosiers",
+    "linkedin": "jason-desrosiers",
+    "email": "jason@hyperjump.io",
+    "type": "tsc",
+    "incorporatedIn": ["California, USA"],
+    "bio": "Jason is a recognized expert and active contributor to the JSON Schema community with over a decade of experience. As a member of the Technical Steering Committee (TSC), he has helped shape the specification and establish best practices. In addition, Jason has developed innovative techniques for writing complex schemas that are scalable, maintainable, efficient, and produce error messages that are clear and helpful.\n\nAs the author and maintainer of the `@hyperjump/json-schema` validator, Jason has created an implementation that is widely regarded as a trusted reference for understanding and verifying correct behavior in complex scenarios.\n\nJason is also highly knowledgeable in using JSON Schema and JSON Hyper-Schema to design APIs that are easy to use, well-documented, and adaptable over time. He helps teams build APIs that not only meet immediate needs but also ensure smooth evolution as business requirements change, making it easier to maintain and scale them as the organization grows.\n\nIn addition to his JSON Schema expertise, Jason holds a Master's degree in Software Engineering and brings extensive experience in Pair/Mob Programming, Test-Driven Development, and coaching teams to improve their coding skills and productivity. He specializes in hands-on collaboration, working alongside teams to teach practical skills in the context of their real-world projects.\n\nIf you need expert guidance on JSON Schema, API design, schema development, or improving your team’s development practices, reach out to Jason to discuss your needs.",
```

Comment:
Great bio!

> Jason has created an implementation that is widely regarded as a trusted reference for understanding and verifying correct behavior in complex scenarios

I confirm! I have used the online playground SO much when I was learning JSON Schema :D